.gitignore

```diff
@@ -69,8 +69,9 @@ typings/
 .yarn-integrity
 
 # dotenv environment variables file
-.env
-.env.test
+#*.env
+#*.env.test
+#*.env.production
```

Comment:
I think we can keep the .env files in the .gitignore for now

Reply:
Ok, I'll add that back